js/kafka/kafka-producer.js

```diff
@@ -80,6 +84,11 @@ export class KafkaProducer extends Producer {
             ),
             messages: this._serializeMessage(message)
         });
+
+        const globalDiffusion =
+            options.globalDiffusion === undefined ? this.globalDiffusion : options.globalDiffusion;
+        const globalTopic = KafkaClient.getGlobalTopic(topic);
+        if (globalDiffusion && globalTopic) await this.produce(globalTopic, message, options);

```

Comment:
I would remove this logic from the Kafka producer. I think a better place would be here: https://github.com/ripe-tech/ripe-bus-api-js/blob/master/js/base.js#L53

Basically, we do this logic there and if it is meant to globally diffuse we call produce again but for the new topic name.